util/install.sh

```diff
@@ -492,9 +496,9 @@ function ryu {
     cd ryu
 
     # install ryu
-    sudo pip install -r tools/pip-requires -r tools/optional-requires \
+    sudo -H ${PYTHON} -m pip install -r tools/pip-requires -r tools/optional-requires \
```

Comment:
Good catch. Ryu is deprecating python2 (but still supporting it for now) and we do want to install with the appropriate $(PYTHON). I haven't looked like this recently but we might be able to just pip install ryu.

Reply:
`pip install ryu` is `python -m pip install ryu`. In a python3 environment we should be using `pip3` instead of `pip` and at the end of the day `pip3`is `python3 -m pip` (actually, I'm more comfortable when calling python3...

Using `sudo -H ${PYTHON}` we tie the use of `pip`or `pip3` to the Python version selected by the installer...